cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -398,6 +398,11 @@ def _should_auto_attach(ua_section: dict) -> bool:
     # pylint: enable=import-error
 
     try:
+        result = util.log_time(
+            logfunc=LOG.debug,
+            msg="Checking if the instance can be attached to Ubuntu Pro",
+            func=should_auto_attach,
+        )
         result = should_auto_attach()
```

Comment:
Were you intending to call `should_auto_attach()` twice here?

Reply:
ups, removed.